Source/DafnyLanguageServer/Workspace/DafnyDocument.cs

```diff
@@ -32,6 +34,35 @@ public record DafnyDocument(
     /// </summary>
     public string? SerializedCounterExamples { get; init; }
 
+
+    /// <summary>
+    /// True is the resolution succeeded, false if resolution failed
+    /// <c>null</c> If the verification did not start (e.g. because of resolution errors)
+    /// </summary>
+    public bool? ResolutionSucceeded { get; set; } = null;
```

Comment:
I need this on the VSCode level to know if the gutter is displayed in gray (resolution not succeeded) or regular.